File: android/app/src/main/java/com/reymelin/gradientclock/widget/TimeFormatter.java
package com.reymelin.gradientclock.widget;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

public class TimeFormatter {
  private static final SimpleDateFormat fmt = new SimpleDateFormat("h:mm", Locale.getDefault());
  public static String now() {
    return fmt.format(new Date());
  }
}

File: android/app/src/main/java/com/reymelin/gradientclock/widget/TimeFormatter.kt
package com.reymelin.gradientclock.widget

import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

object TimeFormatter {
  private val fmt = SimpleDateFormat("h:mm", Locale.getDefault())
  fun nowHHmm(): String = fmt.format(Date())
}
